lib/index.js

```diff
@@ -80,7 +80,10 @@ function inline(it, keyword, schema) {
         }
         _validation.value = _validation.value.map(function (val) {
             const _val = { now: val.now, $data: val.$data, manipulate: val.manipulate };
```

Comment:
does this line throw when `val` is a string?

Reply:
![Screenshot_20200207-230358~2](https://user-images.githubusercontent.com/2249342/74069653-2d141380-49ff-11ea-9195-0c6c6d290d8d.png)

Not at all! :) See the attached screenshot (sorry I am from my mobile away from the PC right now).
The reason is that strings are basically objects. You will get all values set to undefined, but safe enough.